public/locales/de/translation.json

```diff
@@ -134,6 +134,7 @@
       "upload": "Upload"
     },
     "maxworkers": "Maximale Anzahl von gleichzeitigem Herunterladen",
+    "offlinemode": "Spiel Offline Ausführen",
```

Comment:
Change this to `Spiel offline ausführen`

Reply:
Fixed, but I'm curious... What's the difference?

Reply:
I think because we should not capitalize every word. For me doesn't make any difference, but in some languages maybe is not ideal I guess, or not common maybe.

Reply:
> I think because we should not capitalize every word. For me doesn't make any difference, but in some languages maybe is not ideal I guess, or not common maybe.

Yes, you're right. For example, in Russian, no one writes like that, because it's simply wrong.

Reply:
Alright got it. Thanks.

Reply:
> Fixed, but I'm curious... What's the difference?

Just capitalization rules in the language.

> I think because we should not capitalize every word. For me doesn't make any difference, but in some languages maybe is not ideal I guess, or not common maybe.

I also noticed that in the English translation. I would probably change this here and there.